packages/synapse-react-client/src/components/FeaturedDataTabs/FeaturedDataTabs.tsx

```diff
@@ -54,23 +56,30 @@ const FeaturedDataTabs: React.FunctionComponent<
       {/* tab content */}
       {selectedTabProps && (
         <>
-          <FeaturedDataPlots
-            key={`${sql}-${selectedTabIndex}`}
-            {...selectedTabProps.plotsConfig}
-            rgbIndex={rgbIndex}
-            sql={sql}
-            explorePagePath={selectedTabProps.explorePagePath}
-          />
-          {selectedTabProps.explorePagePath && (
-            <div className="bootstrap-4-backport FeaturedDataTabs__explore-all">
-              <Button
-                variant="secondary"
-                size="lg"
-                href={selectedTabProps.explorePagePath}
-              >
-                View All {selectedTabProps.exploreObjectType}
-              </Button>
-            </div>
+          {selectedTabProps.plotsConfig.configs.length > 0 ? (
```

Comment:
If there are no plot configurations, then show a NoContentAvailable UI (wrapped in a Paper)